src/test/apps.test.js

```diff
@@ -115,22 +115,65 @@ describe("Apps", function () {
       AppsActions.requestApps();
     });
 
-    describe("App", function () {
+    describe("basic app", function () {
```

Comment:
we're still using the value `"BASIC"` in `app.type`. Let's keep the code internally consistent.